fileio/output.go

```diff
@@ -26,7 +26,7 @@ const rating_ch_size = 100 // Size of the buffer for the URL channel
 type Rating struct {
 	NetScore       float64 `json:"NetScore"`
 	Url            string  `json:"URL"`
-	License        bool    `json:"License"`
+	License        float64 `json:"License"`
```

Comment:
@emilejbm switched this back so we could output the license subscore as a number